bin/sandbox

```diff
@@ -0,0 +1,8 @@
+# Hooks will not work correctly due to linked modules from sandbox unless the react version is linked:
```

Comment:
Basically, I added this because previously, trying to run the development sandbox just wouldn't work when we're using hooks.

This does some necessary package linking magic in order to make it work.

Should be seamless for the developer, they'll just run `npm run sandbox` and no even know it happened.